regression/expected/react-icu/translationJobs/en_de/Grandfather_en_de_job_xxx0xxx-done.json

```diff
@@ -8,119 +8,87 @@
 			"q": 70,
 			"src": "Welcome to react using react-i18next",
 			"tgt": "Willkommen zu react und react-i18next",
-			"ts": 1653323478091
-		},
-		{
-			"guid": "BrxAEiGogfP3tM7gdyDV8ZyWRYxTFt55z9C-jkwTGEs",
-			"q": 70,
-			"src": "To get started, edit <1>src/App.js</1> and save to reload.",
-			"tgt": "Um loszulegen, ändere <1>src/App.js</1> speicheren und neuladen.",
-			"ts": 1653323478091
+			"ts": 1653363932437
+		},
+		{
+			"guid": "cghma_NsjtZInytTsem0CwlHb1dS2hT2uQGVfDLNkaI",
+			"q": 70,
+			"nsrc": [
+				"To get started, edit ",
+				{
+					"t": "bx",
+					"v": "<1>"
```

Comment:
I'm puzzled why a `<1>` placeholder words. That's not valid xml and it's not defined in i18next afaik.